### DISCUSSION TITLE
`devtools` is an overdependency

### DISCUSSION THREAD

**Discussion by mfansler** [GreenleafLab/ArchR#2233] [Feature Requests]
v1.0.3 adds a `devtools` dependency. However, it is only used to provide `install_github`. That functionality has being spun off to the `remotes` package. Otherwise, `devtools` is intended for those developing R packages, which should not be the case for most imaginable use cases for installing this package.

This package should depend only on `remotes` and remove the `devtools` dependency.

**Comment by immanuelazn** [comment]
Good to know, I'll add in a fix!
